### DISCUSSION TITLE
akhq logging

### DISCUSSION THREAD

**Discussion by red8rain** [tchiotludo/akhq#956] [Q&A] [answered]
hello,

i'm currently running akhq as a background task on aws ec2 using: sudo java -Dmicronaut.config.files=/path/to/application.yml -jar akhq.jar & .

when i kill exit the ssh session, the process is still running but when something happen to akhq, I dont have any insight to what caused it to stop working.  How should I go about setting akhq so that it would log to a file in a specific folder?

thanks.

**Comment by tchiotludo** [comment, marked as answer]
The most simple solution : 
```bash 
sudo java -Dmicronaut.config.files=/path/to/application.yml -jar akhq.jar  > out.log 2>error.log & .
```
